packages/nimble-components/src/rich-text-mention/users/index.ts

```diff
@@ -24,10 +21,19 @@ declare global {
 /**
  * Rich Text user mention configuration element which will have MappingMentionUser elements as children
  */
-export class RichTextMentionUsers extends RichTextMention<
-RichTextMentionConfig,
-RichTextMentionUsersValidator
-> {
+export class RichTextMentionUsers extends RichTextMention<RichTextMentionUsersValidator> {
+    // TODO this isn't a great way to do this. It's mixing configuration element state and parent state which are decoupled asynchronously from each other.
```

Comment:
I'm guessing this was a TODO based on @rajsite's contribution this weekend. It sounds like there is some more work here. Is that happening in this PR or tracked elsewhere?

Reply:
Removed `getMentionedHrefs()` method from the configuration base.  The client will be getting all the mentionedHrefs from the editor/viewer.